solr/solrj/src/test/org/apache/solr/client/solrj/response/TestSpellCheckResponse.java

```diff
@@ -37,14 +40,18 @@ public class TestSpellCheckResponse extends EmbeddedSolrServerTestBase {
 
   @BeforeClass
   public static void beforeClass() throws Exception {
-    initCore();
+    solrClientTestRule
+        .build()
+        .setSolrHome(Paths.get(SolrJettyTestBase.legacyExampleCollection1SolrHome()))
+        .init();
   }
 
   static String field = "name";
 
+  SolrClient client = getSolrClient();
```

Comment:
[0] I don't think it's _wrong_ per se, but I'd rather see this in a method (maybe at the end of `beforeClass` or in a separate `@Before` annotated method?

Not a strong opinion though; if there's a reason you've done it this way then ignore me.

Reply:
Agreed on putting this at the end of beforeClass